scripts/test_single.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from argparse import ArgumentParser
+from load_modules import download_code2seq
+
+download_code2seq()
```

Comment:
While it seems kind of awkward, in the future we can resolve it with a proper `__init__.py`